docker-compose.yml

```diff
@@ -1,9 +1,14 @@
 version: "2"
 services:
+  price:
+    build: https://github.com/GaloyMoney/price.git#main
```

Comment:
This should be a tagged image not `build: ...#main`. This file is not for developing the images but to provide deps _as they are in production_ to develop _this_ app.

Reply:
we are not doing versioning yet on the price pod

Reply:
is there a way to resolve it from chart repository(gcr.io/galoyapp/price)? or it is different?

Reply:
> is there a way to resolve it from chart repository(gcr.io/galoyapp/price with latest tag)? or is it different?

yes you can use the hash. should be shown in the build done by github action

Reply:
I tried with `build: gcr.io/galoyapp/price:latest` and `build: gcr.io/galoyapp/price#latest` but is not working